draft-ietf-httpbis-resumable-upload.md

```diff
@@ -324,7 +324,7 @@ The client SHOULD NOT automatically retry if a client error status code between
 
 Upload appending is used for resuming an existing upload.
 
-The request MUST use the `PATCH` method and be sent to the upload resource. The `Upload-Offset` field value ({{upload-offset}}) MUST be set to the resumption offset.
+The request MUST use the `PATCH` method and `application/partial-upload` media type and be sent to the upload resource. The `Upload-Offset` field value ({{upload-offset}}) MUST be set to the resumption offset.
```

Comment:
I feel that this sentence is a bit off, probably due to language barriers on my end. 

Suggested edit:
>  The request MUST use the `PATCH` method with `application/partial-upload` as content type and be sent to the upload resource. The `Upload-Offset` field value ({{upload-offset}}) MUST be set to the resumption offset.